StartEmailValidationProcess/handler.ts

```diff
@@ -99,13 +107,29 @@ export function StartEmailValidationProcessHandler(
     );
 
     const dfClient = df.getClient(context);
-    await dfClient.startNew(
-      "EmailValidationProcessOrchestrator",
-      undefined,
-      emailValidationProcessOrchestartorInput
-    );
-
-    return ResponseSuccessAccepted();
+    return taskEither
+      .of(makeStartEmailValidationProcessOrchestratorId(fiscalCode, email))
+      .chain(orchId =>
+        isOrchestratorRunning(dfClient, orchId).chain(_ =>
+          _.isRunning
+            ? taskEither.of(void 0)
+            : tryCatch(
+                () =>
+                  dfClient.startNew(
+                    "EmailValidationProcessOrchestrator",
+                    orchId,
+                    emailValidationProcessOrchestartorInput
+                  ),
+                toError
+              )
+        )
+      )
+      .mapLeft(err => {
+        context.log.error(`${logPrefix}|ERROR=${String(err)}`);
+        throw new Error(String(err));
```

Comment:
I'd remove this part since any error here is already logged with the function name